scripts/userstyles.yml

```
@@ -622,6 +622,14 @@ userstyles:
     readme:
       app-link: "https://minesweeper.online"
     current-maintainers: [*gitmuslim]
+  namemc:
+    name: NameMC
+    categories: [game, entertainment]
```

```suggestion
    categories: [game_development, analytics, discussion_forum]
```

This feel "more right" to me. It isn't a game itself, nor does it provide entertaniment per se. Game *development* seems more apt here, analytics since it tracks accounts in the game, and discussion forum since people can post/upload/share skins and stuff. I'm also now realizing we may want to adjust the categories of Planet Minecraft as well :)